src/module/endpoints/Endpoints.ts

```diff
@@ -0,0 +1,51 @@
+import {App} from "../../ManagementConsole";
+import "../../services/dmr/DmrService";
+import {EndpointConfigCtrl} from "./../endpoints/config/EndpointConfigCtrl";
+
+const module: ng.IModule = App.module("managementConsole.endpoints", []);
+
+
+module.config(($stateProvider: ng.ui.IStateProvider) => {
+
+  $stateProvider.state("edit-endpoint-config", {
+    parent: "root",
+    url: "endpoints/:profile/:endpointType/:endpointName/config",
```

Comment:
You need to create a route, say "endpoints" which implements IRedirectState and points to a child route in the `redirectTo` field. So in this case, "edit-endpoint-config" would become "endpoints.edit-endpoint.config" (you could just call it "endpoints.edit" for short). See the [CacheContainer](https://github.com/infinispan/infinispan-management-console/blob/master/src/module/cache-container/CacheContainer.ts#L23-L40) for an example.